hazelcast/src/test/java/com/hazelcast/osgi/CheckDependenciesIT.java

```diff
@@ -53,15 +53,17 @@
 
             // with the "javax" package we have to be more specific - do not use just "javax."
             // as it contains e.g. javax.servlet which is not part of the SE platform!
+            "javax.annotation",
             "javax.crypto",
+            "javax.jms",
```

Comment:
The JMS API is not part of the Java SE. It shouldn't be whitelisted.

Reply:
Removed it from the whitelist and make its resolution optional in the ee pr: https://github.com/hazelcast/hazelcast-enterprise/pull/4003/commits/a17d30cb4e0f8370ee7b96430cfbf4f91d9909c9